### ISSUE TITLE
[EpicLoot] Feature Request: Andvaranaut ping-radius upgrade

### ISSUE THREAD

**Issue by mparsakia** [RandyKnapp/ValheimMods#199]
Hey all,

I've been running a private server with a few friends using this mod and one of the players had some feedback about the andvaranaut and treasure map system.

Currently the treasure chests can spawn sometimes inside rocks, trees, which makes the andvaranaut useful, however they had a complaint that the radius for the "ping" to take effect is too close, as a result by the time the ring pings they've already found the chest.

As a suggestion, perhaps there could be an upgrade for the andvaranaut (say, consuming coins or some magicmats) that can increase it's detection range to be more in-line with how the wishbone pings.

Best. Mike

**Comment by Digitalroot** [comment]
The value for this is here https://github.com/RandyKnapp/ValheimMods/blob/main/EpicLoot/Adventure/TreasureMapChest.cs#L70

It might be of value to make it a configurable item

**Comment by mparsakia** [comment]
@Digitalroot  Thank you for the info. Unfortunately I have limited experience with games dev, and I am only going through the configs myself. If I knew how to compile this library and change the values I would attempt to.

I think there should be an unenchanted Andvaranaut; currently I spawn the unenchanted version through the console so we can get a +200 carry weight buff (custom legendary config in magiceffects.json) and get rid of the Megingjord. I understand not all players would enjoy this, so this is our workaround for now, but I would appreciate if the unenchanted andvaranaut was added as an alternative to the "Silver Ring" and "Ruby Ring" etc for purposes of imbuing.
